src/Services/Flight/src/Flight/Flights/Dtos/FlightDto.cs

```diff
@@ -2,6 +2,8 @@
 
 namespace Flight.Flights.Dtos;
 
```

Comment:
For Dtos, we shouldn't use value objects! Please revert it to normal case. `(We only use value objects in Aggregate write model)`